CASMNET-2176/README.md

```diff
@@ -0,0 +1,37 @@
+# CASMNET-2176 - Unbound resiliency
+
+## Prerequisites
+
+- CSM 1.4.0 or higher
```

Comment:
Is this also needed for 1.5 and newer, or should this explicitly state it is for CSM 1.4.X?

Reply:
It's in >=1.5.0 but to your point it's not clear that this is aimed at >=1.4.0 and <1.5

Reply:
This fix is only for CSM 1.4.0, 1.4.1, 1.4.2, and 1.4.3. It's included in all later versions.

I stated that later in the readme but moved that up to the prerequisites section.